tests/e2e/c/dynamic_fees.go

```diff
@@ -37,7 +37,10 @@ var _ = e2e.DescribeCChain("[Dynamic Fees]", func() {
 
 	ginkgo.It("should ensure that the gas price is affected by load", func() {
 		ginkgo.By("creating a new private network to ensure isolation from other tests")
-		privateNetwork := e2e.Env.NewPrivateNetwork()
+		privateNetwork := &tmpnet.Network{
+			Owner: "avalanchego-e2e-dynamic-fees",
+		}
+		e2e.Env.NewPrivateNetwork(privateNetwork)
```

Comment:
Hm the naming seems pretty weird here. I wouldn't expect a "New" function to not return a new value - but instead initialize the provided value.